src/Jupyter/Visualization/StateDisplayEncoders.cs

```diff
@@ -15,6 +15,15 @@
 
 namespace Microsoft.Quantum.IQSharp.Jupyter
 {
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum PhaseDisplayStyle
+    {
+        None,
```

Comment:
Similarly, you may want to provide API documentation comments for each enum option.
```suggestion
        /// <summary>
        ///     Suppress phase information.
        /// </summary>
        None,
        /// <summary>
        ///     Display phase information as an arrow (<c>↑</c>) rotated by an angle
        ///     dependent on the phase.
        /// </summary>
```